book/crud.rst

```diff
@@ -0,0 +1,767 @@
+            }) }}
+        {% endif %}
+
+        {% if resource_granted('DELETE', vehicle) %}
+            {{ UI.deleteButton({
+                'dataUrl': path('inventory_api_delete_vehicle', {'id': vehicle.id}),
+                'dataRedirect': path('inventory.vehicle_index'),
+                'aCss': 'no-hash remove-button',
+                'id': 'btn-remove-vehicle',
+                'dataId': vehicle.id,
+                'entity_label': 'Vehicle'|trans,
+            }) }}
+        {% endif %}
+    {% endblock navButtons %}
+
+    {% block pageHeader %}
+        {% set breadcrumbs = {
+            'entity':      form.vars.value,
+            'indexPath':   path('inventory.vehicle_index'),
+            'indexLabel': 'Vehicles'|trans,
+            'entityTitle': vehicle.model
+        } %}
+        {{ parent() }}
+    {% endblock pageHeader %}
+
+    {% block content_data %}
+        {% set data %}
+            <div class="widget-content">
+                <div class="row-fluid form-horizontal">
+                    <div class="responsive-block">
+                        {{ UI.renderProperty('Model'|trans, vehicle.model) }}
+                        {{ UI.renderProperty('Seats'|trans, vehicle.seats) }}
+                        {{ UI.renderProperty('Bought at'|trans, vehicle.boughtAt|date) }}
+                        {{ UI.renderProperty('Leased until'|trans, vehicle.leasedUntil|date) }}
+                    </div>
+                </div>
+            </div>
+        {% endset %}
+        {% set dataBlocks = [
+            {
+                'title': 'Data'|trans,
+                'class': 'active',
+                'subblocks': [
+                    { 'data' : data }
+                ]
+            }
+        ] %}
+
+        {% set id = 'vehicleView' %}
+        {% set data = { 'dataBlocks': dataBlocks } %}
+        {{ parent() }}
+    {% endblock content_data %}
+
+As you can see the blocks being used are basically the same as in the template that was used to
+display a :ref:`form to create and update vehicles <book-crud-create-update>`. Refer to that
+section for an explanation of the different blocks.
+
+.. _book-crud-delete:
+
+Deleting Entities
+-----------------
+
+As you may have noticed, the datagrid configuration used the special ``delete`` type when defining
+the action that will be performed when the user clicks the trash icon for an entry. When the
+``delete`` is used, Oro will perform a ``DELETE`` HTTP request to the referenced resource. You can
+easily create a REST controller that handles the ``DELETE`` request by extending the
+:class:`Oro\\Bundle\\SoapBundle\\Controller\\Api\\Rest\\RestController` class:
+
+.. code-block:: php
+    :linenos:
+
+    // src/InventoryBundle/Controller/Api/Rest/VehicleController.php
+    namespace InventoryBundle\Controller\Api\Rest;
+
+    use FOS\RestBundle\Controller\Annotations\NamePrefix
+    use Oro\Bundle\SoapBundle\Controller\Api\Rest;
+
+    /**
+     * @NamePrefix("inventory_api_")
```

Comment:
Hi, Yes, you are correct here. It should be `inventory_api_` since you are describing CRUD in a stand-alone bundle which is not part of Oro. It should work just fine.